metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -948,7 +948,7 @@ def create(cls, config_dict: dict, ctx: PipelineContext) -> Source:
         return cls(ctx, config)
 
     def get_workunits(self) -> Iterable[MetadataWorkUnit]:
-        if self.server is None:
+        if self.server is None or not self.server.is_signed_in():
```

Comment:
We should add an error somewhere otherwise it will be hard to debug if it just returns without anything. Ideally we would this debugging information in the Source Report like https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source_report/sql/snowflake.py#L15 as that is usually shared by people asking for help.

Reply:
@anshbansal the error is logged already earlier in the code if server is not reachable or if sign in fails.